docs/release/release-team-onboarding.md

```diff
@@ -0,0 +1,61 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Onboarding Notes for Release Team Shadows](#onboarding-notes-for-release-team-shadows)
+  - [Overview](#overview)
+    - [Release Lead Shadows Team](#release-lead-shadows-team)
+    - [Comms Lead Shadows Team](#comms-lead-shadows-team)
+    - [CI Signal Lead Shadows Team](#ci-signal-lead-shadows-team)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Onboarding Notes for Release Team Shadows
+
+## Overview
+
+Welcome to the Cluster API Release Team Shadows onboarding documentation!
+
+As a member of the Release Team and Shadow, you play a crucial role in helping to ensure project releases
+are delivered smoothly and in a timely manner. While there are specific responsibilities for each sub-team within
+the Release Team Shadows, below are some general notes that every member of the Releease Team might benefit by going 
+through at the beginning of the cycle:
+
+- Slack:
+    - Most discussions related CAPI Release topics happens in the #cluster-api channel on the Kubernetes Slack. If you need access to the Kubernetes Slack, please visit http://slack.k8s.io/.
+- Kubernetes SIG membership:
+    -  Try to become an official member of the Kubernetes SIG, if possible. More information on the membership and requirements can be found [here](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-team.md#cluster-api-release-team-vs-kuberneteskubernetes-sig-membership).
+- Familiarize yourself with the Release Process:
+    - Review the release [tasks document](../release/release-tasks.md) which explains the responsibilities and tasks for each role within the release team.
+- Check the Release Timeline:
+    - Go through the [release timeline](../release/releases) of the release cycle you are involved in (i.e checkout `release-1.6.md` if you are part of the 1.6 cycle release team) to better understand the key milestones and deadlines.
+
+Now, let's dive into the specific onboarding notes for each sub-team below.
+
+### Release Lead Shadows Team
```

Comment:
```suggestion
### Release Lead Team
```

Reply:
We should probably have a better name for this team - Release Management Team? And then the Release Lead is a single person.

Reply:
Out of scope for this change though.

Reply:
> We should probably have a better name for this team - Release Management Team? And then the Release Lead is a single person.

I am not sure if coming up with a name at all specifically for the Release Lead Shadows is a good idea since we don't have it for other sub-teams (CI/Comms). 

 I'll leave this open for now, and if we decide to introduce a new team name we can do it in a separate issue/PR.

Reply:
It's just that CI team, and comms team are clear, and we have CI leads and comms leads.

 Release lead team isn't as clear as we also have a single role which is release lead.

Reply:
I'll create a new issue to discuss this one, as it is not directly related to this change